docs/guides/conda.rst

```diff
@@ -8,16 +8,171 @@ and need them installed when building their documentation.
 
 This work was funded by `Clinical Graphics`_ -- many thanks for their support of Open Source.
 
+.. _Clinical Graphics: https://www.clinicalgraphics.com/
+
 Activating Conda
 ----------------
 
 Conda support is available using a :doc:`../config-file/index`, see :ref:`config-file/v2:conda`.
 
-This Conda environment will also have Sphinx and other build time dependencies installed.
-It will use the same order of operations that we support currently:
+Our :ref:`Docker images <builds:Docker images>` use Miniconda, a minimal conda installer.
+After specifying your project requirements using conda ``environment.yml`` files,
+Read the Docs will create the environment (using ``conda env create``)
+and add the core dependencies needed to build the documentation.
+
+Creating the ``environment.yml``
+--------------------------------
+
+There are several ways of `exporting a conda environment`_:
+
+- ``conda env export`` will produce a complete list of all the packages installed in the environment
+  with their exact versions. This is the best option to ensure reproducibility,
+  but can create problems if done from a different operative system than the target machine.
+- ``conda env export --from-history`` will only include packages that were explicitly requested,
+  excluding the transitive dependencies. This is the best option to maximize cross-platform compatibility.
+- And finally, you can also write it by hand. This allows for more fine-tuned control and overcomes
+  some limitations in the conda exporting capabilities.
```

Comment:
I think this is _our_ preferred way and I suppose it's the one we want to recommend somehow. Why? Because otherwise, people install _all the requirements_ to _use the project_, instead only the requirements to _build the documentation_ --which could be minimal in comparison.

I know that there are lot of people using notebooks on their own documentation showing how their own package works. Those case are special and may want to treat them differently.

I'm not sure if this work is in the scope of this PR, tho. Probably is not, but worth to mention it.

Reply:
Probably just adding a note like:  "Create an environment file only with the packages needed to build your docs for faster builds"

Reply:
And/or also, in the second method: "notice that this might include packages that are not needed to build your docs". Thanks for the suggestions!

Reply:
Done, have a look